.gitignore

```diff
@@ -99,16 +103,16 @@ MANIFEST
 # to keep the files generated during the source distribution
 # creation. However, that is more complicated, so we are just
 # going to gitignore these files.
-extern/region-4.9/src/reglexer.c
-extern/region-4.9/src/regparser.c
-extern/region-4.9/src/regparser.h
-extern/region-4.9/src/regparser.tab.h
-extern/region-4.9/src/regparser.tab.c
-extern/region-4.9/src/reglexer.c.tmp
+extern/region-4.11/src/reglexer.c
+extern/region-4.11/src/regparser.c
+extern/region-4.11/src/regparser.h
+extern/region-4.11/src/regparser.tab.h
+extern/region-4.11/src/regparser.tab.c
+extern/region-4.11/src/reglexer.c.tmp
 
 # Sphinx documentation
 docs/_build
-**/api
+docs/**/api
```

Comment:
@DougBurke I had to make this change because the clause would shadow some fftw code. I'd like to make sure this is fine with you before merging. I am not sure what directories it was meant to ignore.

Reply:
I tried the sphinx build, and I believe my change is ignoring the correct folders, i.e. after a build:

````
[2] $ git status --ignored
On branch upgrade-deps
Your branch is up to date with 'myfork/upgrade-deps'.

Ignored files:
  (use "git add -f <file>..." to include in what will be committed)

        __pycache__/
        build/
        docs/data/api/
        docs/fit/api/
        docs/mcmc/api/
        docs/model_classes/api/
        docs/optimisers/api/
        docs/overview/api/
        docs/plots/api/
        docs/statistics/api/
        docs/ui/api/

````

Reply:
Sorry for dropping the ball on this. This change looks good (ie it was meant to ignore any of the api/ directories within docs/).